### DISCUSSION TITLE
Why is the Bard-API output differing greatly from web version?

### DISCUSSION THREAD

**Discussion by gabepen** [dsdanielpark/Bard-API#212] [General]
I'm using bard-api to automate literature searches and asking for links to the articles used to verify certain information. I tested my prompts on the website version and got consistent promising results with accurate and appropriate source links, but when passing the exact same prompts through the api in a python script I am getting inaccurate results and links to completely irrelevant?

The version are that different I guess but is this level of variation expected?

**Comment by dsdanielpark** [comment]
Hello. The current Bard being served on the web is integrated with various services (such as GPT extensions). Furthermore, it seems to be conducting A/B testing targeting various customers, so there could be significant differences depending on your account, country, region, and more. However, for some simple prompts, the results are expected to be quite similar.

In particular, Google, being based on a search engine, is likely to provide better results on the web.

1) Very simple prompt (Please explain about apples.)
2) Prompt involving integration with various services, like Google Flights (Please find me a flight from New York to San Francisco for next week and recommend affordable flight options.)

Could you compare these two and possibly share the results?
